### DISCUSSION TITLE
Add Matter to Sonoff ZigBee Bridge Pro

### DISCUSSION THREAD

**Discussion by cconde** [arendst/Tasmota#21629] [Ideas]
Hello, 

I recently purchased and tasmotized a Bridge Pro which I also planned to use as Matter bridge for my other devices. 

I was surprised it was compiled without Matter support. 

Is there any specific reason for that? Would you consider adding it? 

I know I technically could do a custom compile, but I have to say it looks daunting to do and maintain. Maybe I should go through that rabbit hole but it would be great if it came with standard build for Bridge Pro. 

Thanks!

**Comment by s-hadinger** [comment]
The bridge between Zigbee and Matter is not yet there. I suppose this is the main reason
